packages/@ember/-internals/glimmer/lib/helpers/if-unless.ts

```diff
@@ -156,57 +159,72 @@ export function inlineIf(_vm: VM, { positional }: Arguments) {
   the second argument will be displayed, otherwise, the third argument will be
   displayed
 
-  For example, if `useLongGreeting` is false below:
+  For example, if you pass a falsey `useLongGreeting` to the `Greeting` component:
+
+  ```app/templates/application.hbs
+  <Greeting @useLongGreeting={{false}} />
+  ```
 
-  ```handlebars
-  {{unless useLongGreeting "Hi" "Hello"}} Ben
+  ```app/components/greeting.hbs
+  {{unless @useLongGreeting "Hi" "Hello"}} Ben
   ```
 
   Then it will display:
 
   ```html
-  Hi
+  Hi Ben
   ```
 
-  You can use the `unless` helper inside another helper as a subexpression.
-  If isBig is not true, it will set the height to 10:
+  ## Block form
 
-  ```handlebars
-  {{! If isBig is not true, it will set the height to 10.}}
-  <SomeComponent @height={{unless isBig "10" "100"}} />
-  ```
+  Like the `if` helper, the `unless` helper also has a block form.
 
-  or
+  The following:
 
-  ```handlebars
-  {{some-component height=(unless isBig "10" "100")}}
+  ```app/templates/application.hbs
+  <Greetings />
   ```
 
-  ## Block form
-
-  Like the `if` helper, `unless` helper also has a block form.
-
-  ```handlebars
-  {{! If greetings are found, the text below will not render.}}
+  ```app/components/greetings.hbs
   {{#unless greetings}}
```

Comment:
Doesn't look like this is a block param, so it should either be `@greetings` or `this.greetings`. 

```suggestion
  {{#unless this.greetings}}
```